api/types.proto

```diff
@@ -514,6 +514,15 @@ message TaskStatus {
 	// HostPorts provides a list of ports allocated at the host
 	// level.
 	PortStatus port_status = 6;
+
+	// AppliedBy gives the node ID of the manager that applied this task
+	// status update to the Task object.
+	string applied_by = 7;
```

Comment:
Non-blocking question - more information is always helpful for debugging, but what is this intended to be used for?